test/websocket.test.js

```diff
@@ -2040,6 +2041,29 @@ describe('WebSocket', () => {
         });
       });
     }).timeout(4000);
+
+    describe('tls connect options', () => {
+      const tlsConnect = tls.connect;
+      let tlsConnectOptions;
+
+      beforeEach(() => {
+        tlsConnectOptions = {};
+        tls.connect = (options) => {
+          Object.assign(tlsConnectOptions, options);
+        };
+      });
+
+      afterEach(() => {
+        tls.connect = tlsConnect;
+      });
+
+      it('allows to use empty string to disable sending the SNI extension', () => {
+        const ws = new WebSocket('wss://127.0.0.1', {
```

Comment:
Can you please mock `tls.connect()` here and remove the `describe` wrapper and the `beforeEach()` and `afterEach()` hook? We will add them if/when needed. Thank you.